.github/workflows/gh-pages.yml

```diff
@@ -82,3 +82,34 @@ jobs:
       - name: Push
         run: |-
           git push origin gh-pages:gh-pages
+
+  benchmark:
+    name: Benchmark
+    needs: publish
+    if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: '16'
```

Comment:
Probably should be the "min supported", but also wonder if we can/should matrix this? (12, 14, 16, 18)

Reply:
Since 12 is EOL and we are about to deprecate support, I'll make this 12 for now? The action doesn't support matrixing out of the box but we can look into it.